index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet"> 
+    <title>My Portoflio</title>
+</head>
+<body>
+    <header>
+        <h4 id="welcome-text">Welcome</h4>
```

Comment:
- [ ] [OPTIONAL] I think it can be a good idea to have the text welcome inside an anchor `<a>` instead of `<h4>` since it is considered as the logo and it has to be clickable 😉

Reply:
Thanks for the suggestion. I'll make this change. Have a nice coding or reviewing :smile: